src/index.html

```diff
@@ -1,17 +1,179 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head> 
+    <title>document</title>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    
+    <fieldset class="group_first_field">
+      <legend>Personal information</legend>
+      <div class="input_first_field">
+        <label for="surname_field">Surname:</label>
+        <input 
+            id="surname_field" 
+            type="text"
+            name="surname"
+            autocomplete="off"
+        >
+      </div>
+      <div class="input_field">
+        <label for="name_field">Name:</label>
+        <input 
+            id="name_field" 
+            type="text"
+            name="name"
+            autocomplete="off"
+        >
+      </div>
+      <div class="input_field">
+        <label for="age_field">How old are You?</label>
+        <input
+            id="age_field"
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+        >
+      </div>
+      <div class="input_field">
+        <label for="birth_field">Full date of birth:</label>
+        <input 
+            id="birth_field"
+            type="date"
+            name="birthday"
+            min="1900-01-01"
+            max="2022-12-27"
+        >
+      </div>
+      <div class="input_last_field">
+        <label for="checkbox_field">I accept the term of the agreement  
+        </label>
+        <input 
+            id="checkbox_field"
+            type="checkbox"
+            name="terms"
+        >
+      </div>
+    </fieldset>
+    <fieldset class="group_field">
```

Comment:
Add empty line between _multiline_ siblings
```suggestion
    </fieldset>
    
    <fieldset class="group_field">
```
Apply all over the page